server_code/historic_server/persistence.py

```diff
@@ -20,25 +21,44 @@
 LOGGER = logging.Logger("historic-persistence", level=logging.INFO)
 
 
-class Authorization:
+def _default_identifier():
+    try:
+        user = anvil.users.get_user()["email"]
+    except Exception as e:
+        if isinstance(e, TypeError) or repr(e).startswith("ServiceNotAdded"):
```

Comment:
Oh again!

I've just added `ServiceNotAdded` to anvil.server
It was already available on the client. But not on the server!

TODO - when this goes to anvil production we can adjust this exception